pkgs/development/tools/profiling/systemtap/default.nix

```diff
@@ -0,0 +1,56 @@
+{ fetchurl, linuxPackages, makeWrapper, runCommand, fetchgit
+, elfutils, pkgconfig, gettext, stow, gnumake }:
+let
+  ## fetchgit info
+  url = git://sourceware.org/git/systemtap.git;
+  rev = "51b7cae3023adf137081059c7cc44a13f652ba4c";
+  sha256 = "18m3lf11r99f4yr1m172548lghc0i22zqbys1fwlwakbczz0a2lz";
+
+  inherit (linuxPackages) kernel;
+  version = kernel.stdenv.lib.substring 0 6 rev;
```

Comment:
Version strings should satisfy `new > old`. The guidelines now recommend using the commit date in ISO 8601 format.